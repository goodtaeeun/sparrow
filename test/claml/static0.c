static int static_f() { return 0; }
static int static2_f() { return 0; }

int non_static_f() { return 0; }

struct S {
  int (*fp)(void);
};

struct S s = {static_f};
