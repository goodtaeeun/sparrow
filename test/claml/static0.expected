/* Generated by CIL v. 1.7.4 */
/* print_CIL_Input is false */

#line 6 "static0.c"
struct S {
  int (*fp)(void);
};
#line 1 "static0.c"
static int static_f(void) {

  {
#line 1
    return (0);
  }
}
#line 2 "static0.c"
static int static2_f(void) {

  {
#line 2
    return (0);
  }
}
#line 4 "static0.c"
int non_static_f(void) {

  {
#line 4
    return (0);
  }
}
#line 10 "static0.c"
struct S s = {&static_f};
